frontend/components/commons/results/ResultsList.vue

```diff
@@ -77,9 +101,23 @@ export default {
     },
     isLastPagePaginable() {
       if (this.dataset.results.total > this.paginationLimit) {
-        return (this.dataset.viewSettings.pagination.page * this.dataset.viewSettings.pagination.size) === this.dataset.viewSettings.pagination.maxRecordsLimit;
+        return (
+          this.dataset.viewSettings.pagination.page *
+            this.dataset.viewSettings.pagination.size ===
+          this.dataset.viewSettings.pagination.maxRecordsLimit
+        );
+      }
+      return false;
+    },
+    metadata() {
+      if (this.visibleRecords.find(i => i.id === this.showMetadata)) {
```

Comment:
Could we just keep the whole record item as `this.showMetadata` and fetch metadata from it instead find it in all visible records?